app/src/androidTest/java/com/example/sharingang/WishlistViewTest.kt

```diff
@@ -52,7 +52,7 @@ class WishlistViewTest {
         onView(withId(R.id.addToWishlist)).check(matches(withText(R.string.remove_wishlist)))
         onView(withId(R.id.addToWishlist)).perform(click())
         onView(isRoot()).perform(pressBack())
-
+        Thread.sleep(1000)
```

Comment:
Why is this necessary?

Reply:
For some reason when removing the buttons, when the list was updating, it would find 2 items with the same id/text and the test would crash. I'm just waiting for the list to update before continuing.

Reply:
Interesting; gotta do what you gotta do